### PR TITLE
Set vehicle time when finishing initial parameters read

### PR DESCRIPTION
**Pull request by patrickelectric**
This PR need ardupilot/ardupilot#7716 modification.

### CODE REVIEW

File: src/Vehicle/Vehicle.cc [hunk 1]
```diff
@@ -2132,12 +2132,32 @@ void Vehicle::_rallyPointLoadComplete(void)
...
 {
+    // Try to set current unix time to the vehicle
+    _setSystemTime();
     if (parametersReady) {
         _setupAutoDisarmSignalling();
         _startPlanRequest();
     }
 }
 
+void Vehicle::_setSystemTime(void)
```
[the remark this answers is not in the document]
Comment (DonLakeFlyer) [reply src/Vehicle/Vehicle.cc]: This new stupid github review thing makes me crazy. These comments have been sitrting for days!

**Comment by DonLakeFlyer** [comment]
@patrickelectric Review changes requested

**Comment by patrickelectric** [comment]
I'm waiting https://github.com/ArduPilot/ardupilot/pull/7716 review to merge.

**Comment by DonLakeFlyer** [comment]
If you need to wait for a pull to go through ArduPilot first then how is this a safe change for any older ArduPilot firmware version which does not have the changes from that pull?

**Comment by patrickelectric** [comment]
It's not a question of safeness, but of time, my objective is to use with #7716 and add the capability in QGC for that. Besides, I'm working over others projects that have a high priority over this PR. I can use my free time in the weekend to do the necessary corrections, but until there I'll be not able.

**Comment by DonLakeFlyer** [comment]
Understood.

**Comment by patrickelectric** [comment]
Done, sorry for take this long !

**Pull request merged**